content/docs/04.5-devx/08-registries/00-helm-registry.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Helm Registries"
+metaTitle: "Palette Dev Engine for Enterprise Developers"
+metaDescription: "Palette Dev Engine Helm Registries"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Palette Helm Registry
+Helm Charts are a collection of Kubernetes resource files capable of deploying services ranging in varying complexities. Palette provides a few stable public Helm registries out of the box. Tenants can also add any public or private Helm registries to leverage charts. Palette makes deploying typical Kubernetes applications easier with native support for Helm charts. We also allow the addition of any public or private Helm chart registry to the system. Charts from these system registries or any additional ones added by tenant administrators are modelled into cluster profiles for cluster deployment.
+
+Palette mandates using Helm OCI version 3.7.0 and above for the best security experience.
+
+#  Deploying Helm Charts
+Palette allows the deployment of Helm charts in two modes, one using the default Public Repo and another leveraging developers's custom Helm registry.
```

Comment:
In Palette users can either configure their own repos and use the charts out of the preconfigured out of box helm repos like Bitnami.